task-manager-backend/src/mail/mongo/mail-mongo.js

```
@@ -0,0 +1,71 @@
+const ApiError = require("../../utils/api-error");
+const Mail = require("./mail-model");
+const {
+    MAIL_CREATE_FAILED,
+    MAIL_FETCH_FAILED,
+    MAIL_UPDATE_FAILED,
+    MAIL_DELETE_FAILED,
+} = require("../../commons/constants");
+
+const createMail = async (data) => {
+    try {
+        return await Mail.create(data);
+    } catch (error) {
+        throw new ApiError(500, MAIL_CREATE_FAILED);
```

The status code of 500 should not be thrown.  you should just throw the error and handle in controller. please do it in all places

---

resolved